src/moonstone-samples/lib/All/All.js

```diff
@@ -160,7 +160,7 @@ module.exports = kind({
 					{kind: Button, content: 'Back', small: true, href: 'index.html', mixins: [LinkSupport]}
 				],
 				components: [
-					{name: 'list', kind: DataList, fixedChildSize: 63, components: [
+					{name: 'list', kind: DataList, fixedChildSize: 62, components: [
```

Comment:
I believe in enyo/list we have that optimization/option.  We could conceive of an 'auto' option, but we will need to rework VerticalDelegate in any case.